s2-site/examples/react-component/pagination/demo/pivot.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { SheetComponent } from '@antv/s2-react';
+import '@antv/s2-react/dist/style.min.css';
+
+fetch(
+  'https://gw.alipayobjects.com/os/bmw-prod/2a5dbbc8-d0a7-4d02-b7c9-34f6ca63cff6.json',
+)
+  .then((res) => res.json())
+  .then((dataCfg) => {
+    const s2Options = {
+      width: 600,
+      height: 480,
+      pagination: {
+        pageSize: 4,
+        current: 1,
+      },
+    };
+
+    ReactDOM.render(
+      <SheetComponent
+        dataCfg={dataCfg}
+        options={s2Options}
+        showPagination={true}
```

Comment:
```suggestion
        // showPagination={true}
        showPagination={{
           onChange: (current) => {
              console.log(current)
           },
           onShowSizeChange: (pageSize) => {
              console.log(pageSize)
           }
        }}
```

顺便加上 antd 透传回调的示例? 说明可传 boolean 值, 也可以传回调, 下同